### PR TITLE
Travel fund application: FOSS4G Ilya Boyandin

### PR DESCRIPTION
**Pull request by ilyabo**
I submitted a talk about [flowmap.gl](http://flowmap.gl) to the FOSS4G conference and the [proposal was accepted](https://talks.osgeo.org/foss4g-2023/talk/review/CPDDPVUGSXUSTQ3JKS3H3ZWWUEQBSNSK), so I am planning to go there. Would be great if I could get support from OpenJS for the trip.

Request for: [FOSS4G Prizren, Kosovo](https://2023.foss4g.org/)
Dates: 28-30 June
Travelling from: Zurich
Registration fee: 490 EUR (or 350 EUR early bird)
Flight: ~400 EUR
Hotel: ~300 EUR
Total: ~1200 EUR

cc @openjs-foundation/cpc
cc @bensternthal

### CODE REVIEW

File: programs/travel-fund/2023.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
-| Lea | Verou | TAG F2F | TAG Member | TBD | Tokyo, Japan | April 17-21 | $3500 | March 27th | TBD | TBD | TBD | TBD | TBD | TBD |
-| Dylan | Schiemann | OpenJS World | Program Committee & yoga session | TBD | Vancouver | 8th May - 13th May | 2376 USD | March 30th | https://github.com/openjs-foundation/community-fund/pull/18 | TBD | TBD | TBD | TBD | TBD |
-| Jordan | Harband | OpenJS World | attendance + Collab Summit | TBD | Vancouver | 8th May - 16th May | 1445 CAD (hotel) + $387.10 USD (flight) + ~$100 USD (transportation) | March 6th | https://github.com/openjs-foundation/cross-project-council/pull/1019 | TBD | TBD | TBD | TBD | TBD |
+| First Name | Last Name | Event                              | Reason                                                                                            | Trip Report | Location        | Travel Dates       | Amount Requested                                                     | Pull Request date | Pull Request link                                                    | Date Expense report sent | Amount of Expense Report | Date Sent to Finance | Date approved through Bill.com | Bill.com Amount approved for reimbursement |
```
[the remark this answers is not in the document]
Comment (ilyabo) [reply programs/travel-fund/2023.md]: @ljharb Sorry, missed that, should be good now

**Review by joesepi — APPROVED** [review]
LGTM after formatting is resolved

File: programs/travel-fund/2023.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ljharb) [reply programs/travel-fund/2023.md]: (my advice is to disable prettier and any autoformatting in your editor, unless a project has specifically enabled it)

File: programs/travel-fund/2023.md [hunk 2]
```diff
@@ -7,3 +7,4 @@
 | Lea | Verou | TAG F2F | TAG Member | TBD | Tokyo, Japan | April 17-21 | $3500 | March 27th | TBD | TBD | TBD | TBD | TBD | TBD |
 | Dylan | Schiemann | OpenJS World | Program Committee & yoga session | TBD | Vancouver | 8th May - 13th May | 2376 USD | March 30th | https://github.com/openjs-foundation/community-fund/pull/18 | TBD | TBD | TBD | TBD | TBD |
 | Jordan | Harband | OpenJS World | attendance + Collab Summit | TBD | Vancouver | 8th May - 16th May | 1445 CAD (hotel) + $387.10 USD (flight) + ~$100 USD (transportation) | March 6th | https://github.com/openjs-foundation/cross-project-council/pull/1019 | TBD | TBD | TBD | TBD | TBD |
+| Ilya       | Boyandin  | [FOSS4G](https://2023.foss4g.org/) | [giving a talk](https://talks.osgeo.org/foss4g-2023/talk/review/CPDDPVUGSXUSTQ3JKS3H3ZWWUEQBSNSK) | TBD         | Prizren, Kosovo | 28-30 June         | EUR 490 registration fee + EUR ~400 flight + EUR ~300 hotel          | April 4th         |                                                                      | TBD                      | TBD                      | TBD                  | TBD                            | TBD                                        |
```
[the remark this answers is not in the document]
Comment (ilyabo) [reply programs/travel-fund/2023.md]: Removed the unnecessary spaces. Is it better now?

**Review by ovflowd — APPROVED** [review]
LGTM!

**Review by christian-bromann — APPROVED** [review]

**Pull request merged**